### PR TITLE
8299617: CurrencySymbols.properties is missing the copyright notice

### PR DESCRIPTION
**Pull request by justin-curtis-lu**
Added the missing copyright header

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8299617](https://bugs.openjdk.org/browse/JDK-8299617): CurrencySymbols.properties is missing the copyright notice

### Reviewers
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**)
 * [Iris Clark](https://openjdk.org/census#iris) (@irisclark - **Reviewer**)
 * [Jaikiran Pai](https://openjdk.org/census#jpai) (@jaikiran - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11854/head:pull/11854` \
`$ git checkout pull/11854`

Update a local copy of the PR: \
`$ git checkout pull/11854` \
`$ git pull https://git.openjdk.org/jdk pull/11854/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11854`

View PR using the GUI difftool: \
`$ git pr show -t 11854`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11854.diff">https://git.openjdk.org/jdk/pull/11854.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties [hunk 1]
```diff
@@ -1,3 +1,28 @@
+#
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```
Comment (naotoj) [inline test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties:2]: This file is not new, so I'd use `2001` as the year created, which is borrowed from the test body `CurrencyFormat.java`.

File: test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
```
Comment (naotoj) [inline test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties:9]: Classpath exception is not applicable to tests.

**Review by naotoj — CHANGES_REQUESTED** [review]

File: test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties (hunk 2 above)
Comment (justin-curtis-lu) [reply test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties]: Did not know that, thank you. Made the fixes for this and your previous comment.

**Review by irisclark — APPROVED** [review]
Provided GPL license appear to match the standard template available here:

https://github.com/openjdk/jdk/blob/master/make/data/license-templates/gpl-header

**Review by naotoj — APPROVED** [review]

**Review by jaikiran — APPROVED** [review]

**Pull request closed without merging**